### PR TITLE
Feature/jhipster 5.7.2

### PR DESCRIPTION
**Pull request by kkyrychenko**

### CODE REVIEW

File: .jhipster/Task.json [hunk 1]
```diff
@@ -1,73 +0,0 @@
-{
-    "fluentMethods": true,
-    "relationships": [],
-    "fields": [
-        {
```
Comment (v-kyrychenko) [inline .jhipster/Task.json]: please keep this file

File: README.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
-[![Build Status](https://travis-ci.org/xm-online/xm-ms-scheduler.svg?branch=master)](https://travis-ci.org/xm-online/xm-ms-scheduler) [![Quality Gate](https://sonarcloud.io/api/project_badges/measure?&metric=sqale_index&branch=master&project=xm-online:xm-ms-scheduler)](https://sonarcloud.io/dashboard/index/xm-online:xm-ms-scheduler) [![Quality Gate](https://sonarcloud.io/api/project_badges/measure?&metric=ncloc&branch=master&project=xm-online:xm-ms-scheduler)](https://sonarcloud.io/dashboard/index/xm-online:xm-ms-scheduler) [![Quality Gate](https://sonarcloud.io/api/project_badges/measure?&metric=coverage&branch=master&project=xm-online:xm-ms-scheduler)](https://sonarcloud.io/dashboard/index/xm-online:xm-ms-scheduler)
```
Comment (v-kyrychenko) [inline README.md]: keep this section

File: src/main/java/com/icthh/xm/ms/scheduler/config/WebConfigurer.java [hunk 3]
```diff
@@ -20,34 +28,38 @@
...
-
-import java.nio.charset.StandardCharsets;
-import java.util.*;
-import javax.servlet.*;
 
 /**
  * Configuration of web application with Servlet 3.0 APIs.
  */
-@Slf4j
-@RequiredArgsConstructor
```
Comment (v-kyrychenko) [inline src/main/java/com/icthh/xm/ms/scheduler/config/WebConfigurer.java]: keep this changes

File: src/main/resources/logback-spring.xml [hunk 4]
```diff
@@ -61,8 +61,6 @@
     <logger name="LiquibaseSchemaResolver" level="INFO"/>
     <logger name="sun.net.www" level="INFO"/>
     <logger name="sun.rmi.transport" level="WARN"/>
-    <logger name="org.reflections" level="INFO"/>
-    <logger name="org.apache.kafka.clients" level="ERROR"/>
```
Comment (v-kyrychenko) [inline src/main/resources/logback-spring.xml]: do not remove this log levels

**Review by v-kyrychenko — CHANGES_REQUESTED** [review]

**Review by v-kyrychenko — APPROVED** [review]

File: .yo-rc.json [hunk 5]
```diff
@@ -1,18 +1,17 @@
...
+      "packageName": "com.icthh.xm.ms.scheduler"
     },
-    "jhipsterVersion": "4.13.2",
+    "jhipsterVersion": "5.7.2",
+    "applicationType": "microservice",
     "baseName": "scheduler",
     "packageName": "com.icthh.xm.ms.scheduler",
     "packageFolder": "com/icthh/xm/ms/scheduler",
-    "serverPort": "8081",
+    "serverPort": "8086",
```
Comment (amedvedchuk) [inline .yo-rc.json:11]: why do we need to change port?

File: src/main/java/com/icthh/xm/ms/scheduler/config/DatabaseConfiguration.java [hunk 6]
```diff
@@ -48,22 +50,23 @@
...
      * @return the H2 database TCP server
      * @throws SQLException if the server failed to start
      */
-    @Bean(initMethod = "start", destroyMethod = "stop")
-    @Profile(JHipsterConstants.SPRING_PROFILE_DEVELOPMENT)
-    public Object h2TCPServer() throws SQLException {
-        log.debug("Starting H2 database");
-        return H2ConfigurationHelper.createServer();
-    }
+//    @Bean(initMethod = "start", destroyMethod = "stop")
```
Comment (amedvedchuk) [inline src/main/java/com/icthh/xm/ms/scheduler/config/DatabaseConfiguration.java]: remove if it is not needed

File: src/main/resources/config/application-dev.yml [hunk 7]
```diff
@@ -40,34 +45,40 @@ spring:
...
-        console:
-            enabled: true
+        url: jdbc:postgresql://localhost:5432/scheduler
+        username: postgres
+        password: postgres
     jpa:
-        database-platform: io.github.jhipster.domain.util.FixedH2Dialect
-        database: H2
-        show-sql: true
+        database-platform: io.github.jhipster.domain.util.FixedPostgreSQL82Dialect
```
Comment (amedvedchuk) [inline src/main/resources/config/application-dev.yml]: why do you change H2 DB for test to Postgres?

File: .yo-rc.json (hunk 5 above)
Comment (v-kyrychenko) [reply .yo-rc.json:11]: according to the
https://github.com/xm-online/xm-online/wiki/Ports-reservation-map

8086 | Microservice for Scheduler | xm-ms-scheduler
-- | -- | --

File: src/main/java/com/icthh/xm/ms/scheduler/config/DatabaseConfiguration.java (hunk 6 above)
Comment (v-kyrychenko) [reply src/main/java/com/icthh/xm/ms/scheduler/config/DatabaseConfiguration.java]: fixed

File: src/main/resources/config/application-dev.yml (hunk 7 above)
Comment (v-kyrychenko) [reply src/main/resources/config/application-dev.yml]: fixed

**Review by ishkurko — APPROVED** [review]

**Review by amedvedchuk — APPROVED** [review]

**Pull request merged**